src/buildFuseList.js

```diff
@@ -21,7 +21,7 @@ async function buildList() {
   const fuseTokens = await fetchBridgedTokens(foreignAddresses);
   const bridgedFuseTokens = mainnet.map(mainnetToken => {
     const fuseToken = fuseTokens.find(token => token.foreignAddress == mainnetToken.address.toLowerCase());
-    if (fuseToken) {
+    if (fuseToken && fuseToken.symbol !== 'FUSE') {
```

Comment:
@mul53 not liking too much the way you filter fuse by symbol.
We need to exclude this token: https://explorer.fuse.io/tokens/0x714005Da6A90F59dD2cF74560ecF4A4BeD5F088A, so doing this by address is IMO most safe